zerobuf/Zerobuf.h

```diff
@@ -47,6 +47,17 @@ class Zerobuf
     /** @return the size of the zerobuf. */
     ZEROBUF_API size_t getZerobufSize() const;
```

Comment:
To avoid namespace pollution. These methods leak into the client code, and it is very likely that someone will have a 'size : uint64_t' in a .fbs schema which will generate a getSize() method.